src/ops.rs

```diff
@@ -214,6 +260,14 @@ pub trait OpTrait {
     fn other_output(&self) -> Option<EdgeKind> {
         None
     }
+
+    /// Get the number of non-dataflow multiports.
+    ///
+    /// If None, the operation must have exactly one non-dataflow port
```

Comment:
Erm.....aaiieeee....

If the operation has other_edges, "None" means "exactly one non-dataflow port". Is that different from `None` meaning `Some(1)` ?

And if the operation does not have other edges, "None" means "zero". Is that different from `None` meaning `Some(0)`?

i.e. can we turn this Option into a usize?